CONTRIBUTORS.yaml

```diff
@@ -617,10 +624,6 @@ reginaesinamabotsi:
     name: Regina Esinam Abotsi
     joined: 2018-06
 
-rosiealice:
-    name: Rosie Fisher
-    joined: 2020-10
-
```

Comment:
It looks like Rosie got accidentally removed, and is breaking the build. Could you please re-add them?

Reply:
Poor Rosie ;) Nice catch eagle eye Helena !

Reply:
Oh shoot ! Looks like I did an unfortunate ctrl+D ... thanks ! :)

Reply:
I re added Rosie this morning